public/js/bio-link/bio-link.js

```
@@ -0,0 +1,215 @@
+        ALLOW_DATA_ATTR: false
+    });
+}
+
+// --- Shared global state ---
+let bioLinks = [];
+let currentBioLink = null;
+let bioLinkItems = [];
+
+// --- Authenticated API call helper ---
+async function apiCall(url, options = {}) {
+    const token = await getAuthToken();
```

ESLint

[error] 'getAuthToken' is not defined.

(no-undef)